core/src/price_estimation.rs

```diff
@@ -115,6 +115,16 @@ impl PriceOracle {
             }
         }
     }
+
+    async fn eth_token_id(&self) -> Result<TokenId> {
+        for token_id in self.token_info_fetcher.all_ids().await? {
+            let info = self.token_info_fetcher.get_token_info(token_id).await?;
+            if info.symbol() == "ETH" {
```

Comment:
Should we allow "WETH" too here? 
Also just a note to ourselves that this is one of the places that will need to be adjusted if we deploy on the xDAI chain where the native gas token is DAI.

In the Ganache tests this falls back to the hardcoded prices since we don't have an ETH token there I assume?

Reply:
In token_info.rs we have an override so that WETH is turned into ETH.

Reply:
> we have an override so that WETH is turned into ETH.

Right, maybe it's still a bit safer to not rely on this here?